pkgs/applications/editors/bluej/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, makeWrapper, jetbrains }:
+
+stdenv.mkDerivation rec {
+  pname = "bluej";
+  version = "4.2.2";
+  src = fetchurl {
+    url = "https://www.bluej.org/download/files/BlueJ-linux-${builtins.replaceStrings ["."] [""] version}.deb";
+    sha256 = "sha256-XCJB8iCOmPz5qtfHooK88W5v1UP6pf2wuZs00QIxE8M=";
+  };
+
+  buildInputs = [ makeWrapper ];
+  propagatedBuildInputs = [ jetbrains.jdk ];
```

Comment:
No need to use this at all, if it's for sure will be referenced because of the installPhase.